playbook.yml

```
@@ -117,6 +117,15 @@
       loop_control:
         label: "{{ item.1.name }}"
 
+    # Prevent out of zone errors caused by trailing periods
+    - name: Remove period at the end of lines
```

```suggestion
    - name: Remove full stops at the end of lines
```
Sorry, sorry American vs British again 😱